# Message variables for the cubic trajectory coefficients
# a0, a1, a2, a3: coefficients for determing shape of trajectory
# t0, tf: initial and final time values for calculating trajectory
float32 a0
float32 a1
float32 a2
float32 a3
float32 t0
float32 tf